### ISSUE TITLE
OBA 2.x: Add CI support with GitHub Actions

### ISSUE THREAD

**Issue by barbeau** [OneBusAway/docker#19]
As noted in https://github.com/OneBusAway/onebusaway-docker/pull/18, CI builds are now failing due to an issue with postgres that doesn't seem related to any changes in the project. 

For example, this build:
https://travis-ci.org/github/OneBusAway/onebusaway-docker/builds/680766690

..of the latest merge to master fails with:

~~~
Successfully tagged oba-ci:latest
=== building bundle
=== running postgres
Unable to find image 'postgres:10' locally
10: Pulling from library/postgres
Status: Downloaded newer image for postgres:10
893b48792079979360c734a233af641c4dfccfb284014fb15b65ea04e03bf7b1
=== creating postgres database
Error response from daemon: Container 893b48792079979360c734a233af641c4dfccfb284014fb15b65ea04e03bf7b1 is not running
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
Error: No such container: oba-postgres
postgres didn't work
The command "bin/ci" exited with 1.
~~~

**Comment by danp** [comment]
Did some debugging on this in a branch, got this output from the postgres container:

```
Error: Database is uninitialized and superuser password is not specified.
       You must specify POSTGRES_PASSWORD to a non-empty value for the
       superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
       You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
       connections without a password. This is *not* recommended.
       See PostgreSQL documentation about "trust":
       https://www.postgresql.org/docs/current/auth-trust.html
```

Once I did a `docker pull postgres:10` locally I saw it as well. Will PR a fix momentarily.

**Comment by aaronbrethorst** [comment]
fixed in #39
